src/styles/styles.ts

```diff
@@ -3974,6 +3983,117 @@ const styles = (theme: ThemeColors) =>
         singleOptionSelectorCircle: {
             borderColor: theme.icon,
         },
+
+        videoPlayerControlsContainer: {
+            position: 'absolute',
+            bottom: 8,
+            left: 8,
+            right: 8,
+            backgroundColor: '#061B09CC',
+            height: 48,
+            borderRadius: 8,
+            flexDirection: 'column',
+            overflow: 'visible',
+            padding: 4,
+            paddingBottom: 8,
+        },
+
+        videoPlayerControlsButtonContainer: {
+            height: 24,
+            marginBottom: 8,
+            flexDirection: 'row',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+        },
+
+        progressBarOutline: {
+            width: '100%',
+            height: 5,
+            borderRadius: 10,
+            backgroundColor: '#FFFFFF51',
+        },
+
+        progressBarFill: {
+            height: '100%',
+            backgroundColor: colors.white,
+            borderRadius: 10,
+        },
+
+        videoPlayerControlsRow: {
+            flexDirection: 'row',
+            alignItems: 'center',
+        },
+
+        videoPlayerText: {
+            textAlign: 'center',
+            fontSize: 13,
```

Comment:
I'm not sure about this one, but we have similar sizes defined in variables. But please verify, if it makes sense here.
```suggestion
            fontSize: variables.fontSizeLabel,
```